client/src/services/languages.js

```diff
@@ -0,0 +1,53 @@
+//Request handler import
+const request = require("../utils/requestHandlers");
+
+const getLanguages = async (dispatch) => {
+  dispatch({ type: "GET_LANGUAGES" });
+  //Make get call to API for languages
+  const config = {
+    method: "get",
+    url: "/api/languages",
+  };
+
+  const data = await request.getFromAPI();
+  // return { languages: ["Javascript"] };
+
+  if (!data) {
+    dispatch({ type: "GET_LANGUAGES_ERROR" });
+    return "Request Failed";
+  }
+
+  dispatch({ type: "GET_LANGUAGES_SUCCESS" }, { languages: data.data });
```

Comment:
In reducers set up for "GET_LANGUAGES_SUCCESS", it was **user** instead of languages. 

Instead of passing data.data, utilize destructuring eg. languages: data